CodeEdit/Features/CodeFile/CodeFile.swift

```diff
@@ -70,7 +70,7 @@ final class CodeFileDocument: NSDocument, ObservableObject, QLPreviewItem {
         fileURL
     }
 
-    @Published var cursorPosition = (1, 1)
+    @Published var cursorPositions = [CursorPosition]()
```

Comment:
How does this look in our statusbar? If it is now an array, are we showing the cursor position of the last item in the array or handling it in some other way?

Reply:
Looks like we are not, not a problem. I'll do this in a separate PR.

Reply:
Yeah it's on a TODO right now. Hopefully this change helps match Xcode's status bar. `CursorPosition` contains both range and line/column data so we can do the entire range of line/column, multiple selection, or range selection like Xcode does.